guides/common/modules/con_content-view-environment-ordering-and-priority.adoc

```diff
@@ -0,0 +1,58 @@
+[id="content_view_environment_ordering_and_priority_{context}"]
+= Content view environment ordering and priority
```

Comment:
Seems like `RedHat.CaseSensitiveTerms` needs an update so that `content view environment` is a thing.

Reply:
Agree. That particular rule has been downgraded from error to warning because of problems with false positives.